tests/unit_tests/libs/controllers/test_predictor.py

```diff
@@ -334,37 +336,21 @@ def test_house_pricing(self, use_gpu):
                   stop_training_in_x_seconds=1,
                   use_gpu=use_gpu)
 
-        def assert_prediction_interface(prediction):
-            assert hasattr(prediction, 'data')
-            assert hasattr(prediction, 'extra_insights')
-            assert hasattr(prediction, 'transaction')
-            assert hasattr(prediction[0], 'explanation')
-            assert hasattr(prediction[0], 'data')
-            assert prediction[0].predict_columns[0] == 'rental_price'
-
-            for item in prediction:
```

Comment:
Why remove this part? I see that `__getitem__` interface is still there, so why not test it?

Reply:
I've made the members private (i.e. from `member` to `_member`) to indicated that they shouldn't be tested.

There's not point in testing for the existence of private member... even testing for public members is overkill, but I can see the point there, since a user may wish to access them and maybe someone took them out by mistake.